cloudera/hatch.toml

```diff
@@ -0,0 +1,4 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["27", "38"]
```

Comment:
Are we going to support python 2.7 for this integration?

Reply:
Originally I thought we would need to add support for Python 2, but after talking with the team I think I can drop it and only keep Python 3.8.